README.md

```diff
@@ -62,10 +62,10 @@ Check out [plugins documentation](./docs/plugins.md).
 First, clone the repo via git:
 
 ```bash
-git clone https://github.com/KELiON/cerebro.git cerebro
+$ git clone https://github.com/KELiON/cerebro.git cerebro
```

Comment:
You might not want to add a `$` here, because it's easier to copy-paste the command without having to manually remove the prompt character. Most Markdown linters would recommend the same.